bundles/io/org.eclipse.smarthome.io.net/META-INF/MANIFEST.MF

```diff
@@ -31,3 +31,4 @@ Import-Package:
  org.slf4j
 Bundle-ActivationPolicy: lazy
 Service-Component: OSGI-INF/*.xml
+Require-Bundle: org.eclipse.jetty.util

```

Comment:
You could add the following packages in Import-Package header.

```
org.eclipse.jetty.util,
org.eclipse.jetty.util.component,
org.eclipse.jetty.util.ssl
```

Reply:
Just noticed. The packages are already added. I think the Require-Bundle header will not be necessary.

Reply:
Will check this.

Reply:
done